### ISSUE TITLE
.to not working as indicated in the mapping tutorial

### ISSUE THREAD

**Issue by kenphused** [outr/scalarelational#76]
I am following the mapping tutorial and have encountered a compile error because "value to is not a member of org.scalarelational.instruction.Query."  

The gist for the code is:  https://gist.github.com/kenphused/14f7fb9d71a0e9743f96

I may be missing something and appreciate the help.

-Ken

**Comment by darkfrog26** [comment]
What version of ScalaRelational are you using?

**Comment by darkfrog26** [comment]
Change:

```scala
import org.scalarelational.mapper.{MappedTable, Entity}
```

to

```scala
import org.scalarelational.mapper._
```

Everything should work then.  There is a package-level implicit to do mapping.

**Comment by kenphused** [comment]
I am using v 1.1.0 and can confirm the import you suggested works.  Thanks for your quick reply!

**Comment by darkfrog26** [comment]
No problem, let me know if you have any other issues. What documentation are you using?

The latest documentation is available here:

http://scalarelational.org/scalarelational/latest/

**Comment by kenphused** [comment]
I was using the docs at the URL you noted above.  The docs are not explicit for the import. Do you guys need help with documentation?  I am new to functional programming / Scala, but I am glad to help any way I can.

**Comment by darkfrog26** [comment]
We can always use more help.  We've been working towards getting the documentation solidified for a while now, but we don't seem to have enough time to get it finished.

The import is explicitly referenced here: http://scalarelational.org/scalarelational/latest/mapper.html#library-imports-1

**Comment by kenphused** [comment]
Yes, the import is part of the documentation, but for me, it was not clear that the ...mapper._ import was explicitly needed (my IDE filled in the ...mapper. imports and I thought those were sufficient).

Oh...  I just peeked at your profile darkfrog and noticed you live in Norman, OK.  I lived there from 64 - 67...  2758 South Pickard Ave...  we were the first family to live in that house.  The area south of Imhoff Road was a farm "back in the day."
